src/.vuepress/config/nav/en.js

```diff
@@ -11,6 +11,10 @@ module.exports = [
 						text: "Getting started",
 						link: "/help/guides/getting-started/",
 					},
+					{
+						text: "Adding External Repositories",
+						link: "/help/guides/adding-repos/"
+					}
```

Comment:
Probably not needed but there were some coma before.
```suggestion
						link: "/help/guides/adding-repos/",
					},
```